vehicle/saic/requests/encryption.go

```diff
@@ -48,18 +47,18 @@ func CalculateResponseVerification(str, str2, str3 string) string {
 	return HmacSha256(a4, str5)
 }
 
-func EncryptRequest(url string, time int64, tenant, token, body, contentType string) string {
+func EncryptRequest(path string, time int64, tenant, token, body, contentType string) string {
 	sendDate := strconv.FormatInt(time, 10)
 	// tenant
-	replace := ""
-	if len(url) != 0 {
-		replace = strings.Replace(url, BASE_URL_P, "/", -1)
+	resourcePath := ""
+	if len(path) != 0 {
```

Comment:
Is Encrypt ever called with empty path? If not we don't need this check.

Reply:
Probably don't need the check, but would rather leave it incase there is a reason for it.   It doesn't hurt the code. II don't want to spend the time to completely understand the entire SAIC API, just here to add AU Region.  - so minimal changes. Don't want to break something I am not aware of.